src/auth/providers.js

```diff
@@ -289,7 +294,22 @@ function googleProvider(auth) {
      * @param {Object} data the redirectUrl data to use for authentication
      * @returns {Promise} a promise that resolves when authentication succeeds.
      */
-    authenticate: data => {
+    authenticate: async data => {
+      let { authCode } = data;
+      if (authCode !== null) {
```

Comment:
FYI, if `data` is equal to `{ authCode: undefined }` or even just `{}`, this check will be truthy. i.e. `undefined !== null`. might be worth defining something like https://github.com/10gen/stitch-ui/blob/master/src/util.js#L176-L184 in the sdk